From e731f3e28b7e7d1c745b03084e01036ee00018eb Mon Sep 17 00:00:00 2001
From: Daniel Verkamp <dverkamp@chromium.org>
Date: Mon, 12 Nov 2018 15:22:16 -0800
Subject: [PATCH] lib/raid6: add missing include for raid6test
Git-commit: e731f3e28b7e7d1c745b03084e01036ee00018eb
Patch-mainline: v5.0-rc1
References: bsc#1166003

Add #include <sys/time.h> for gettimeofday() to fix the compiler warning
about an implicitly defined functions.

Signed-off-by: Daniel Verkamp <dverkamp@chromium.org>
Signed-off-by: Shaohua Li <shli@fb.com>
Signed-off-by: Coly Li <colyli@suse.de>
---
 include/linux/raid/pq.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/raid/pq.h b/include/linux/raid/pq.h
index ea8505204fdf..0c245dcb8b48 100644
--- a/include/linux/raid/pq.h
+++ b/include/linux/raid/pq.h
@@ -35,6 +35,7 @@ extern const char raid6_empty_zero_page[PAGE_SIZE];
 #include <limits.h>
 #include <stddef.h>
 #include <sys/mman.h>
+#include <sys/time.h>
 #include <sys/types.h>
 
 /* Not standard, but glibc defines it */
-- 
2.25.0

